### PR TITLE
Added wikidata and data tags for Soma and bd’s Mongolian Grill.

### LINKED ISSUE #3055: Soma
`shop=clothes` `clothes=women` `alt_name=Soma Intimates`

https://www.soma.com/
https://en.wikipedia.org/wiki/Chico's_%28clothing_retailer%29#Soma

Needs a separate Wikidata item that can be related to the Chico’s Wikidata item using the [subsidiary](https://www.wikidata.org/wiki/Property:P355) and [subsidiary of](https://www.wikidata.org/wiki/Property:P749) properties. A public domain logo in SVG format can be found in the source code of that webpage – search for `logo-soma` – and uploaded to Wikimedia Commons for use in the Wikidata item.

### PR DESCRIPTION
**Pull request by sinclarius**
Resolves #3055 and #3061.

### CODE REVIEW

**Comment by sinclarius** [comment]
Messed up my commit message 😓 but 46efac1 is the bd’s Mongolian Grill addition.

File: brands/amenity/restaurant.json [hunk 1]
```diff
@@ -2189,14 +2131,21 @@
       "name": "Zizzi"
     }
   },
+  "amenity/restaurant|bd’s Mongolian Grill": {
+    "countryCodes": ["mn", "us"],
+    "matchNames": ["bd Mongolian Grill"],
+    "tags": {
+      "amenity": "restaurant",
+      "brand": "bd’s Mongolian Grill",
+      "brand:en": "bd’s Mongolian Grill",
```
Comment (tas50) [inline brands/amenity/restaurant.json]: We probably don't need the name:en here if it's the same

File: brands/amenity/restaurant.json [hunk 2]
```diff
@@ -2189,14 +2131,21 @@
       "name": "Zizzi"
     }
   },
+  "amenity/restaurant|bd’s Mongolian Grill": {
+    "countryCodes": ["mn", "us"],
+    "matchNames": ["bd Mongolian Grill"],
+    "tags": {
+      "amenity": "restaurant",
+      "brand": "bd’s Mongolian Grill",
```
Comment (tas50) [inline brands/amenity/restaurant.json]: can you replace the ’ with ' here

File: brands/amenity/restaurant.json [hunk 3]
```diff
@@ -2189,14 +2131,21 @@
...
+      "brand:en": "bd’s Mongolian Grill",
+      "brand:wikidata": "Q69890408",
+      "cuisine": "mongolian_bbq",
+      "name": "bd's Mongolian Grill",
+      "name:en": "bd's Mongolian Grill"
```
Comment (tas50) [inline brands/amenity/restaurant.json]: Same here with the :en that's the same

File: brands/shop/clothes.json [hunk 4]
```diff
@@ -2723,6 +2723,18 @@
       "shop": "clothes"
     }
   },
+  "shop/clothes|Soma": {
+    "countryCodes": ["us"],
+    "tags": {
+      "alt_name": "Soma Intimates",
+      "brand": "Soma",
+      "brand:wikidata": "Q69882213",
+      "brand:wikipedia": "en:Soma",
```
Comment (tas50) [inline brands/shop/clothes.json]: This doesn't seem to go to a specific page. If there isn't a wikipedia page you can just leave it out

**Comment by tas50** [comment]
Thanks for adding Soma. I've had this one on my todo list for a while. It's one of the few US mall brands left that we don't have a match for.

File: brands/shop/clothes.json (hunk 4 above)
Comment (sinclarius) [reply brands/shop/clothes.json]: I was a little confused about this—it doesn't link to a Wikipedia page (looks like it's been removed: https://en.wikipedia.org/wiki/BD's_Mongolian_Grill), but I did add a wikidata entry with a logo: https://www.wikidata.org/wiki/Q69890408. The Wikidata page is all we'd need to pull the logo into the nsi, right?

Comment (tas50) [reply brands/shop/clothes.json]: The wikidata page is the important part. Just nuke both of those wikipedia pages and everything will be good for the index.

**Comment by Adamant36** [comment]
Might want to add social links to the Wikidata entry also just for the heck of it as backups for the icon.

**Comment by Adamant36** [comment]
I just merged it and fixed the Wikipedia link in ebc0c41. Thanks @sinclarius

**Pull request merged**